cypress/components/confirm/confirm.cy.tsx

```diff
@@ -200,7 +200,7 @@ context("Testing Confirm component", () => {
     });
 
     it("should check Esc key is disabled", () => {
-      const callback = cy.stub();
+      const callback: ConfirmProps["onCancel"] = cy.stub();
       CypressMountWithProviders(<ConfirmComponent disableEscKey />);
```

Comment:
but it is still not used here like:
```suggestion
CypressMountWithProviders(<ConfirmComponent disableEscKey onCancel={callback} />);
```